crates/api_models/src/admin.rs

```diff
@@ -75,6 +75,10 @@ pub struct MerchantAccountCreate {
     #[cfg(not(feature = "multiple_mca"))]
     #[schema(value_type = Option<PrimaryBusinessDetails>)]
     pub primary_business_details: Option<Vec<PrimaryBusinessDetails>>,
+    ///Will be used to expire client secret after certain amount of time to be supplied in seconds
+    ///(900) for 15 mins
+    #[schema(example = 900)]
+    pub intent_fulfillment_time: Option<i64>,
```

Comment:
Since we don't perform any validations for negative values, can we have it as a `u32` instead? You can infallibly convert it to a an `i64` since there is a `From` implementation. Moreover, it should be a sufficiently large limit for payment intent fulfillment times. Please make this change for the update request as well.

Reply:
We actually had u32 itself but it was creating some issues reagarding DB and diesel conflicts that is why we converted it to BIGINT and i64. What do you suggest should try again is it?